src/bot/actions.js

```diff
@@ -95,7 +96,7 @@ const dripLater = async (sender, address, time) => {
   const { amount } = dripActions[DRIP_TYPE.LATER];
   const { data: { hash, providerId } } = res;
 
-  return `I will send ${amount} NEU to address ${address} at ${time} UTC. Extrinsic hash: ${hash}. Your provided_id: ${providerId}.`;
+  return `I will send ${amount} ${tokenSymbol} to address ${address} at ${time} UTC. Extrinsic hash: ${hash}. Your provided_id: ${providerId}.`;
```

Comment:
this `address` in `to address ${address}` can be removed.

UTC shouldn’t be needed to type in manually in `${time} UTC`. Try this, `moment.utc(value).format("MM-DD HH:mm:ss Z")`, which will translate the timestamp in UTC to a string.

Reply:
Add one more sentence to the success messsage.

`Try out our recurring payment feature on https://ace.web3go.xyz! (Click on Connect at the top right corner to select Turing Staging network)`

Reply:
Fixed